### ISSUE TITLE
STUDY GROUPS: card css could have more height

### ISSUE THREAD

**Issue by stain88** [codebuddies/codebuddies#1098]
Do you want to request a feature or report a bug?
**Bug**

What is the current behavior?
<img width="384" alt="low_height" src="https://user-images.githubusercontent.com/8461452/50575594-2e850c80-0df9-11e9-9819-4a05ed1f2c6c.png">

What is the expected behavior?
Bit more height to avoid the overflow on the bottom.

Which browser / OS are affected by this issue? (An *isolated* way to reproduce the bug)
FF/Safari/Chrome on Mac, so I'm guessing most.

**Comment by stain88** [comment]
As a second tweak, I'd add a `white-space: nowrap;` on the `small` around the user class, to avoid that breaking on two lines:
<img width="377" alt="split_user_count" src="https://user-images.githubusercontent.com/8461452/50575647-d8649900-0df9-11e9-8eec-8da5b47fd8f3.png">

**Comment by lpatmo** [comment]
I don't see a PR related to this but I _think_ it looks fixed on codebuddies.org/groups now (and maybe I forgot to close). Feel free to re-open if I'm wrong!

**Comment by stain88** [comment]
Both issues are still present on all Mac browsers:

<img width="1325" alt="card issues" src="https://user-images.githubusercontent.com/8461452/52901652-23960800-31fe-11e9-9357-37c86dbcbe0d.png">

**Comment by lpatmo** [comment]
D'oh, you're right, not sure how I confused it with some other cards. Sorry!

**Comment by stain88** [comment]
So, "Large" windows are fine, "Small" windows are fine, but the 992-1200px range is still not quite there. "Readings in Computer Science and Technology" seems to be a particularly long one. But at exactly 992px width, this happens:

<img width="1104" alt="992px_cards" src="https://user-images.githubusercontent.com/8461452/52911656-ab335380-329e-11e9-88d2-8c1788303424.png">

**Comment by lpatmo** [comment]
Ah wow, 992px exactly! I had to install a browser width chrome extension to capture that since I kept sliding the browser past 992px.

**Comment by akosasante** [comment]
I'd like to take a look at this issue if it's still occurring 👋

**Comment by stain88** [comment]
Yep, I'm still reproducing this at exactly 992px width, on several browsers.
All yours, @akosasante

**Comment by akosasante** [comment]
I've opened a PR 😄  I may have mishandled my git-fu though, it looks like it brought in a bunch of extraneous commits :/

https://github.com/codebuddies/codebuddies/pull/1143
